src/posit/connect/resources.py

```diff
@@ -129,32 +145,37 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return repr(self.fetch())
 
-    def create(self, **attributes: Any) -> Any:
+    def create(self, **attributes: Any) -> _T:
         response = self._ctx.client.post(self._path, json=attributes)
         result = response.json()
         uid = result[self._uid]
         path = posixpath.join(self._path, uid)
-        return _Resource(self._ctx, path, **result)
+        resource = self._factory(self._ctx, path, **result)
+        resource = cast(_T, resource)
```

Comment:
From other repos, I've used an explicit type expectation before casting the value.  

```python
        resource: _T_co = self._factory(self._ctx, path, **result)
        resource = cast(_T, resource)
```

While it feels redundant, it gives me confidence if we ever change `._factory()` and its return type.  This way, we know we are transferring from a `_T_co` to ` _T` (and not an `Any` to a `_T`).